tests/math/test_functions.py

```diff
@@ -253,6 +254,82 @@ def test_allclose(t1, t2):
     assert res == expected
 
 
+class TestAllcloseSparse:

```

Comment:
Might have missed it, but I assume we have sparse-sparse in here as well as sparse-dense

Reply:
Yes we do